kev_lambda.py

```diff
@@ -1,6 +1,8 @@
 import boto3
 import requests
 import tweepy
+import os
+import json
```

Comment:
If you implement requests.post(url, json=payload), then I believe you can remove this library.